distributed/tests/test_actor.py

```diff
@@ -485,10 +490,7 @@ def check(counter, blanks):
     result = await c.compute(final, actors=counter)
     assert result == 0 + 1 + 2 + 3 + 4
 
-    start = time()
-    while a.data or b.data:
-        await asyncio.sleep(0.01)
-        assert time() < start + 30
+    await async_wait_for(lambda: a.data or b.data, timeout=10)
```

Comment:
I don't think this is the same as the original check since `async_wait_for` negates the value of the `predicate` function passed in:

https://github.com/dask/distributed/blob/ee06a44cc4b43270b5ae6ee102481f3bbc9daf27/distributed/utils_test.py#L1075

Reply:
I am putting this back how it was too, since it doesn't concern my changes.